app/code/Magento/Inventory/Model/ReservationBuilder.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model;
+
+use Magento\InventoryApi\Api\Data\ReservationInterface;
+
+/**
+ * Used to instantiate ReservationInterface objects
+ */
+class ReservationBuilder implements ReservationBuilderInterface
+{
+    /**
+     * @var int
+     */
+    private $reservationId;
+
+    /**
+     * @var int
+     */
+    private $stockId;
+
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var float
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $metadata;
+
+    /**
+     * Object Manager instance
+     *
+     * @var \Magento\Framework\ObjectManagerInterface
+     */
+    protected $_objectManager = null;
```

Comment:
the property should not be protected. 
Just private properties are allowed

Reply:
also, it makes sense to follow the naming convention in M2.2. and call attribute as `$objectManager`
without underscore prefix